crates/spirv-std/macros/Cargo.toml

```diff
@@ -15,4 +15,7 @@ spirv-types = { path = "../shared", version = "0.4.0-alpha.10" }
 heck = "0.3.2"
 proc-macro2 = "1.0.24"
 quote = "1.0.8"
-syn = { version = "1.0.58", features=["full"] }
+syn = { version = "1.0.58", features = ["full"] }
+
+[dev-dependencies]
+spirv-std = { path = ".." }
```

Comment:
I don't think this is worth the extra build time and complexity to validate a single doctest. (I didn't even know one was allowed to have a circular dev-dependency, seems like a thing to avoid.)